### DISCUSSION TITLE
Unable to connect Openedge DB(for Progress DB) getting Driver issues while connecting to Progress DB

### DISCUSSION THREAD

**Discussion by kirankarni** [dbeaver/dbeaver#13129] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version
- DBeaver version
![Openedge Driver issue](https://user-images.githubusercontent.com/20198952/123675297-8c4f0f80-d810-11eb-9097-ac802e1c2b02.PNG)

- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)?

**Comment by kirankarni** [comment]
Hi,
Can you please help me on how to setup progress Db colud setup in DBeaver since I am getting driver issues as shown below error messages
No suitable driver found for jdbc:datadirect:openedge://148.177.26.54:18111;databaseName=mfgdb;EncryptionMethod=ssl;validateServerCertificate=false

**Comment by ShadelessFox** [comment, marked as answer]
Hello @kirankarni,

You have to add required driver files.
